NOTICE-binary

```diff
@@ -1515,3 +1515,16 @@ Copyright 2014-2017 The Apache Software Foundation
 
 This product includes software developed at
 The Apache Software Foundation (http://www.apache.org/).
+
+
+Metrics
```

Comment:
I think this should have already been there, but now would also need to be in `NOTICE`